dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/FdcTestDataCreatorService.java

```diff
@@ -85,6 +85,8 @@ public Set<Integer> createFastTrackTestData(
           testDataClient.createFdcItems(fdcItemBuilder.build());
           fdcItemBuilder = fdcItemBuilder.itemType(FdcItemType.AGFS).adjustmentReason("Pre AGFS Transfer").paidAsClaimed("N").latestCostInd("Current");
         }
+        if (fdcAccelerationType.equals(FdcAccelerationType.PREVIOUS_FDC) && testType.equals(FdcTestType.NEGATIVE_PREVIOUS_FDC))
+          fdcItemBuilder.adjustmentReason("Other");
```

Comment:
I know this should have curly braces. This is the merged branch. I'll do a follow-up PR to reformat this class - there are other formatting issues in the code too.